trunk/src/graphics/paraview/geom2vtk.F90

```diff
@@ -173,9 +213,11 @@ subroutine geom2vtk(Sname,Sfile, IcE,IcN,IcP)
 !
          if (SECOND_ORDER_VIS) then
             GEOM_OBJ(k+1:k+j) = nverl(1:j)
+            if(VIS_VTU) ELEM_TYPES(VTU_element_type_offset(iel) + i) = l
          else
             GEOM_OBJ(k+1) = l
             GEOM_OBJ(k+2:k+1+j) = nverl(1:j)
+            if(VIS_VTU) ELEM_TYPES(VTU_element_type_offset(iel) + i) = l
```

Comment:
if it's the same statement in `if / else`, why not have it outside the branching?

Reply:
Changes Implemented